### DISCUSSION TITLE
How can i build for production.

### DISCUSSION THREAD

**Discussion by tunahan-guler** [web-infra-dev/modern.js#4747] [Q&A]
I extracted the application files by typing yarn build. I put these files in iis but it doesn't work. How do I do this?
The application is MFE sub app!!!!

**Comment by chenjiahan** [comment]
We currently do not have a deployment guide for iis. You can refer to ["output files"](https://modernjs.dev/builder/en/guide/basic/output-files.html) to understand the dist directory structure of Modern.js and how to custom the output structure.

For MFE, the deployment becomes more complex, and the deployment forms of different MFE frameworks may be different.

**Reply by tunahan-guler** [reply]
Can u give me a guide for iis
